packages/hash/api/src/model/knowledge/link.model.ts

```diff
@@ -354,7 +356,7 @@ export default class {
   /**
    * Remove the link.
    *
-   * @param removedbyId - the account removing the link
+   * @param removedbyId - the id of the user removing the link
```

Comment:
I didn't remove `removedById` because it's already a required provenance field on `graphApi.removeLink`. This propagates into some related methods, which is why we have `updatedById` for the `updateWithoutUpdatingSiblings ` method for instance.